src/components/ui/UiChartDonut.vue

```diff
@@ -41,13 +41,40 @@ const chartOptions = ref({
     },
   },
 })
+
+const labelFormatted = computed(() => {
+  const lengthLabelTotal = props.dataGraph.labelTotal.length
+  const part1 = props.dataGraph.labelTotal.slice(0, 10)
```

Comment:
Could you put a sample `labelTotal` here? It's difficult to see what the code does without seeing an expected label or example of it.